Make It Clear Issue Has Been Crowdfunded V2

pinkiebell opened the pull request:
Fixes #2164

pinkiebell commented:
![screenshot 2018-09-07 at 02 09 57](https://user-images.githubusercontent.com/40266861/45201946-8363ac80-b277-11e8-8469-c08888ce2446.png)
![screenshot 2018-09-07 at 08 23 49](https://user-images.githubusercontent.com/40266861/45201955-91193200-b277-11e8-884d-bf4c4ae0b04c.png)

thelostone-mc commented on app/assets/v2/js/shared.js:
```diff
@@ -955,63 +955,107 @@ var usdToAmount = function(event) {
 };
 
-function renderBountyRowsFromResults(results) {
+function renderBountyRowsFromResults(results, renderForExplorer) {
```
❤️

thelostone-mc commented on app/assets/v2/images/coin.svg:
```diff
@@ -0,0 +1,16 @@
+            </g>
+        </g>
+    </g>
+</svg>
```
@pinkiebell can we run `svgo` against this file ?

pinkiebell commented:
@thelostone-mc svgo-ing Done 👍

pinkiebell commented:
Side note: I labelled this pr as wip because of one open question:
  - How do we handle tokens with no usd conversion?
     Right now we have no indicator for that and fixing requires backend changes,
     do we want to handle that case in this PR or defer?

PixelantDesign commented:
@pinkiebell could we display the token with no usd conversion in the tool tip but not average in the final USD value?

pinkiebell commented:
Updates
![screenshot 2018-09-08 at 00 54 17](https://user-images.githubusercontent.com/40266861/45246607-8b1c6300-b302-11e8-8bc5-424c98d535e5.png)
![screenshot 2018-09-08 at 00 54 28](https://user-images.githubusercontent.com/40266861/45246615-8ce62680-b302-11e8-94d0-cbb4355db36f.png)

pinkiebell commented:
The only thing what's left is the caret(arrow) with the right color(same as link color).
I'm also open for another round of improvements 👍

thelostone-mc commented:
@pinkiebell could you rebase it once ? we'll put this up on staging and test it out there today

pinkiebell commented:
@thelostone-mc Done.

Errors still in this PR are likely:
The tooltip misses the 'as of'  <time>.
I'm a little bit confused with the '$' sign in the crowdfunding tooltip,
because I always think of dollar valuation if I see the '$', was that actually right or is this wrong to convert the crowdfunding token value to dollar in the tooltip? 🤔

pinkiebell commented:
@thelostone-mc 
Any news?

SaptakS commented:
@pinkiebell I will be reviewing it in some time and get back to you. :)

SaptakS commented:
@pinkiebell I am seeing this in every non-crowfunded bounty details page. Can you please check?
<img width="523" alt="screen shot 2018-09-24 at 4 04 16 pm" src="https://user-images.githubusercontent.com/9530293/45947896-84386480-c013-11e8-80fc-d3894a5327bb.png">

pinkiebell commented:
@SaptakS
I can't reproduce this behaviour, what wonders me is the 'as ofnow' in the USD tag.
That looks like the JS code is not from this PR. Maybe a cache issue?

@mbeacom 
I addressed your comments. I did a rebase though, hope that's ok.

pinkiebell commented:
@SaptakS
Fun times 😄 . This may be problem with docker and inotify events.
Just reproduced that in the issue explorer because I switched branches/rebased and the django app didn't reload (fs inotify events).

SaptakS commented:
@pinkiebell The icon beside `See All` is opposite according to me. It should point downwards when it is closed and upwards when it is open. cc: @thelostone-mc @mbeacom @PixelantDesign

SaptakS commented on app/assets/v2/js/pages/bounty_details.js:
```diff
@@ -225,60 +221,67 @@ var callbacks = {
 
-        tooltip_info.push(template);
-      }
+    const decimals = 2;
```
We are showing 3 decimal places in almost all places. Any reason why we are showing only 2 for crowdfunded amount?

pinkiebell replied:
@SaptakS Aesthetics? 😄 Fixed!

pinkiebell commented:
@SaptakS Fixed the arrow rotation. But don't ask me why I wrote accordion in the commit message 🤣 .

PixelantDesign commented:
@SaptakS could you put this on staging please?

SaptakS commented:
@PixelantDesign it's on staging at https://stage.gitcoin.co/

PixelantDesign commented:
@pinkiebell could we left align the USD values?

<img width="267" alt="screen shot 2018-09-24 at 10 28 14 pm" src="https://user-images.githubusercontent.com/238235/45989338-4a825080-c049-11e8-9b10-02ae21f1ba9d.png">

pinkiebell commented:
@PixelantDesign 
1. That's the ETH amount in USD, we should probably change the formatting?
2. Did you mean putting the green USD tag on the left or should they all be aligned to the right for example? That could be a little bit tricky with our current CSS if we want to make sure that they are all aligned top-down.

@mbeacom 
Hehe, we can tell future prices, nice! I will put a fix in 👍

pinkiebell commented:
![one](https://user-images.githubusercontent.com/40266861/46048840-c81e8d00-c12b-11e8-8a2f-f499d75c255d.png)
![two](https://user-images.githubusercontent.com/40266861/46048937-2481ac80-c12c-11e8-89ae-f0f590cba16c.png)

pinkiebell commented:
@mbeacon @SaptakS @PixelantDesign 
FInal review? 🙏

thelostone-mc commented:
@pinkiebell one final rebase and then we can throw it on staging

pinkiebell commented:
@thelostone-mc 
Done 👍

PixelantDesign commented:
Looking good to me...One thing I see... 

It should not be possible to have 791 cents.

<img width="241" alt="screen shot 2018-10-01 at 11 11 01 am" src="https://user-images.githubusercontent.com/238235/46298576-66b44f00-c56d-11e8-877e-4c4ff8ac1e7e.png">

pinkiebell commented:
@PixelantDesign 
Resolved 👍 . We now display token amounts with max .3 decimals and dollar conversion .2 decimals.
